src/routes/privacy-policy/+page.svelte

```diff
@@ -0,0 +1,84 @@
+<script lang="ts">
+	import { pageTitle } from '../../stores.js';
+	pageTitle.set('Privacy Policy');
+</script>
+
+<div class="secondary-bg">
+	<div class="section">
+		<h1>Privacy Policy</h1>
+		<p class="emphasis">
+			This privacy notice tells you what to expect us to do with your personal information.
+		</p>
+
+		<ul>
+			<li><a href="#contact-details">Contact details</a></li>
+			<li><a href="#what-information">What information we collect, use, and why</a></li>
+			<li><a href="#lawful-bases">Lawful bases and data protection rights</a></li>
+			<li><a href="#personal-information-source">Where we get personal information from</a></li>
+			<li><a href="#retention">How long we keep information</a></li>
+			<li><a href="#complaints">How to complain</a></li>
+		</ul>
+
+		<h2 id="contact-details">Contact details</h2>
+		<p>Email: <a href="mailto:management@dddsouthwest.com">management@dddsouthwest.com</a></p>
+
+		<h2 id="what-information">What information we collect, use, and why</h2>
+		<p>We collect or use the following personal information to provide updates and share news:</p>
+		<ul>
+			<li>Names</li>
+			<li>Email address</li>
+		</ul>
+
+		<h2 id="lawful-bases">Lawful bases and data protection rights</h2>
+		<p>
+			Under UK data protection law, we must have a “lawful basis” for collecting and using your
+			personal information. There is a list of possible lawful bases in the UK GDPR. You can find
+			out more about lawful bases on the <a href="https://ico.org.uk">ICO’s website</a>.
+		</p>
+		<p>
+			To make a data protection rights request, please contact us using the contact details at the
+			top of this privacy notice.
+		</p>
+
+		<h2>Our lawful bases</h2>
```

Comment:
For consistency (and a11y), we should either make this one a `h3` subheading or add an id and link in the table of contents

Reply:
Good shout!